po/zh-CN.po

```diff
@@ -1236,24 +1244,23 @@ msgstr ""
 "（AOSP）。课程内容包括与 C、C++ 和 Java 的互操作性。"
 
 #: src/index.md
-#, fuzzy
 msgid ""
 "[Chromium](chromium.md): a half-day course on using Rust within Chromium "
 "based browsers. This includes interoperability with C++ and how to include "
 "third-party crates in Chromium."
 msgstr ""
-"[Rust 在 Chromium 中的用法](../chromium.md) 深析是一个为期半天的课程，主要介"
-"绍了如何在 Chromium 浏览器中使用 Rust。课程内容包括如何在 Chromium 的 `gn` 构"
-"建系统中使用 Rust，引入第三方库 (\"crates\")，以及 C++ 的互操作性。"
+"[Chromium](chromium.md)：为期半天的课程，介绍如何在基于 Chromium 的浏览器中使"
+"用 Rust。课程内容包括与 C++ 的互操作性以及如何在 Chromium 中加入第三方 "
+"crate。"
```

Comment:
```suggestion
"用 Rust。课程内容包括与 C++ 的互操作性以及如何在 Chromium 中加入第三方库 "
"（crate）。"
```

Reply:
"库" is the translation of "library" in the [glossary](https://google.github.io/comprehensive-rust/zh-CN/glossary.html), translating "crate" as it is ambiguous.

Since "包" means "package" in [TRPL](https://kaisery.github.io/trpl-zh-cn/ch07-00-managing-growing-projects-with-packages-crates-and-modules.html) but "crate" in [Rust Course](https://course.rs/basic/crate-module/crate.html), it is recommended to leave "crate" untranslated to avoid ambiguity.

Reply:
Thanks for your helpful and elaborate explanation! 

I made this translation to be consistent with an existing translation at [line 1642](https://github.com/google/comprehensive-rust/pull/1800/files#diff-c988d119cac895c3ba265e862e07af0b71f51b99f8fc6642fa550ca4ec0e662fR1642).

Then I will fix it as well😉